apis/profile/v1alpha1/managedclusterprofilebinding_types.go

```diff
@@ -58,6 +58,8 @@ type ManagedClusterProfileBindingStatus struct {
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 	// +optional
 	ObservedOpscenterFeaturesVersion string `json:"observedOpscenterFeaturesVersion,omitempty"`
+	// +optional
+	LastHandledForceUpgradeAt string `json:"lastHandledForceUpgradeAt,omitempty"`
```

Comment:
// +optional
	LastUpgradeAt metav1.Time `json:"lastUpgradeAt,omitempty"`

Why not just this ?